documentation/docs/steps/neoDeploy.md

```diff
@@ -54,3 +54,54 @@ steps:
       account: <myDeployAccount>
       host: hana.example.org
 ```
+
+## Example for invalidating the cache
+
+* Setting parameter `invalidateCache` to `true`, will clean up the cache of the deployed application to portal site and reflects the new changes. This is **applicable** only when the application is deployed and accessed via Portal service (Fiori Launchpad site).
+
+Setting this parameter to true, needs additional configurations to go through. Firstly, create an oauth credential as shown below:
+
+* **Oauth credential creation**:
+
+  * Select the oauth settings from your subaccount, create a new client with subscription to portal/nwc as shown in the following images:
+
+    ![Oauth client creation](../images/oauthClientCreation.png)
+
+  * Select the client tab, which provides an option to register a new client as shown below:
+
+    ![Register new client](../images/registerNewOauthClient.png)
```

Comment:
If 'Clients' were already chosen in the above image (oauthClientCreation.png) this image would be redundant.